src/MyEventEmitter.js

```diff
@@ -1,12 +1,119 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.prependListeners = {};
+    this.commonListeners = {};
```

Comment:
Why did you create two objects for your listeners?

Reply:
I used 2 objects here to distinguish prepend listeners from common ones. If we add all listeners in one object and simply will shift prepend listeners, then the one declared the last will be called the first